### DISCUSSION TITLE
Got 99.6% match even there is no diff, why? (#99 Gift Box)

### DISCUSSION THREAD

**Discussion by programus** [cssbattle/forum#203] [Q&A] [answered]
This is my code: 
```
<i style=background:#AC474B;width:5in;height:4in;margin:-8><i h style=margin:60+175;border-radius:50%+50%+0+50%><i h style=margin:0+30;border-radius:50%+50%+50%+0><i style=margin:50-75;width:60;height:60;background:#fff;box-shadow:80px+0#fff,0+80px#fff,80px+80px#fff><style>i{display:inline-block}i[h]{width:20;height:20;outline:solid+10px#fff}
```

What is the problem? Or just a bug of CSS Battle? 
![image](https://user-images.githubusercontent.com/956125/147401837-6f136931-95f7-45db-ab26-72168580d027.png)

**Comment by rafehimself** [comment, marked as answer]
the problem is that `outline` works like some time ago and do not round corners
@chinchang maybe time for update?

you can see it here: [`https://codepen.io/alexzaworski/embed/vwmNKr`](https://codepen.io/alexzaworski/embed/vwmNKr)
![image](https://user-images.githubusercontent.com/58576132/147406343-26332632-aab2-4185-a98f-6c3ef39543ef.png)

**Reply by programus** [reply]
Thank you for your great answer. I think I got your point. 
The engine used by CSS Battle to render css to image dose not support outline round corners, so there is 0.4% difference while CSS Battle calculate the points. While the UI of CSS Battle I am using just show the rendering result of my browser, so I see a no diff result. 

Well, I will try to use border or box-shadow instead.
